components/dashboard/src/admin/Settings.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext } from "react";
+import { AdminContext } from "../admin-context";
+import CheckBox from "../components/CheckBox";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService } from "../service/service";
+import { adminMenu } from "./admin-menu";
+
+export default function Settings() {
+    const { sendTelemetry, setSendTelemetry } = useContext(AdminContext);
+
+    const actuallySetTelemetryPrefs = async (value: boolean) => {
+        await getGitpodService().server.adminSettingsSetTelemetry(value);
+        setSendTelemetry(value);
+    }
+
+    return (
+        <div>
+            <PageWithSubMenu subMenu={adminMenu} title="Settings" subtitle="Configure settings for your Gitpod cluster.">
+                <CheckBox
+                    title="Send anonymous service ping to Gitpod"
+                    desc="This is used to provide insights on how you use your cluster so we can provide a better overall experience."
```

Comment:
**suggestion**: If we change the title as described above we could go with one of the following proposals depending if we have any relevant docs in place:

Option :a:
```suggestion
                    desc={<span>To help improve Gitpod, service ping will periodically collect anonymous usage information. <a className="gp-link" href="https://www.gitpod.io/docs/LINK-TO-DOCS-OR-PRIVACY-POLICY">Learn more</a></span>}
```

Option :b:
```suggestion
                    desc={<span>To help improve Gitpod, service ping will periodically collect anonymous usage information. Read our <a className="gp-link" href="https://www.gitpod.io/privacy">Read our Privavy Policy</a></span>}
```